### ISSUE TITLE
A11y keyboard control Tab component TabList Tab selection notification

### ISSUE THREAD

**Issue by DaveBest99** [angular/components#3421]
#### Bug, feature request, or proposal:
Bug: Screen reader user is not notified of TabPanel screen refreshes.

#### What is the expected behavior?
When focus is on a Tab in a TabList, with either horizontal or vertical orientation, pressing Spacebar or Enter key should activate the Tab if it was not activated automatically on focus. Upon the Tab selection, the user should be notified of a page content update, either by moving the focus to the top of the TabPanel or providing an aria-alert message. Note, if Tab elements are automatically activated upon focus, then an aria-alert message should be spoken by the screen reader as the user presses Left and right arrow keys, and a Tab key press will move focus to the first TabPanel focusable element.

#### What is the current behavior?
Currently the screen reader says nothing and the focus does not change.

#### Which versions of Angular, Material, OS, browsers are affected?
Testing was performed on Windows10 with JAWS17 and browsers Firefox/IE11/Chrome.

#### Is there anything else we should know?
The Spacebar does not select current TabList Tab element (Enter key does select the element). Also, upon selecting the Tab element, there is no alert message or focus change to notify the user of a screen update. The blind screen reader user has no knowledge that the TabPanel content has refreshed. Although there is no focus change and no spoken alert message, the braille display continues to show the Tab element text label.
